LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023-present Boshen
```

Comment:
https://github.com/oxc-project

> [@boshen](https://github.com/boshen) is the MIT license copyright holder, with publishing rights to [npmjs.com](https://npmjs.com/), [crates.io](https://crates.io/) and [marketplace.visualstudio.com/vscode](https://marketplace.visualstudio.com/vscode).

This is in response to this description of the oxc project